src/nvim/api/ui.c

```diff
@@ -111,14 +139,49 @@ Object ui_try_resize(uint64_t channel_id, Integer width,
   if (width <= 0 || height <= 0) {
     api_set_error(err, Validation,
                   _("Expected width > 0 and height > 0"));
-    return NIL;
+    return;
   }
 
   UI *ui = pmap_get(uint64_t)(connected_uis, channel_id);
   ui->width = (int)width;
   ui->height = (int)height;
   ui_refresh();
-  return NIL;
+}
+
+/// @deprecated
+void ui_try_resize(uint64_t channel_id, Integer width,
```

Comment:
it should be, the wire format doesn't change, and this method didn't have any static return type in latest released version 0.1.4 as #4817 was done later